flowr/examples/hamming/DESCRIPTION.md

```
@@ -0,0 +1,46 @@
+sorted merge (without duplicates) of these three substreams. This is
+inherently self-referential — the stream is defined in terms of itself via
+feedback.
+
+The algorithm maintains a sorted set of candidate numbers. Each iteration
+extracts the smallest candidate (the next Hamming number), then generates
+three new candidates by multiplying it by 2, 3, and 5. Duplicates are
+removed to avoid redundant work (e.g., 6 = 2×3 = 3×2).
+
+Root Diagram
+===
+<a href="root.dot.svg" target="_blank"><img src="root.dot.svg"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)